libc/src/__support/OSUtil/linux/quick_exit.cpp

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
ah, missing header block. Let me go back and `git mv` the .h to this .cpp file, then strip out the header guards.